installer/scripts/omsadmin.sh

```diff
@@ -608,6 +649,13 @@ remove_all()
     # Remove LAD workspace
     WORKSPACE_ID="LAD"
     remove_workspace
+
+    # remove scom workspace
+    ls -l $ETC_DIR | grep -w scom > /dev/null 2> /dev/null
```

Comment:
Do you really want `-l` on the `ls` command? It feels like that gives you a lot more things to hit on that you may not want to match. Could you get by with just `ls $ETC_DIR` and still have `grep` match it?